source/MaterialXRenderGlsl/GLTextureHandler.cpp

```diff
@@ -56,21 +56,23 @@ bool GLTextureHandler::bindImage(ImagePtr image, const ImageSamplingProperties&
     }      
     _boundTextureLocations[textureUnit] = image->getResourceId();
 
+    GLenum targetType = image->getHeight() > 1 ? GL_TEXTURE_2D : GL_TEXTURE_1D;
```

Comment:
I assume these target type changes are pulled from the CM code changes
I made ? I don't think they are necessary for any of the test changes here.
It may not be good to add this in as it's only for CM 1D LUTs I added this in.

Reply:
Yes these were part of CM changes, but I thought it would be a good change to add to the texture handler to support 1D textures. I've reverted it. I'll be removing CM changes from contrib so wanted to align the main with contrib.